test/run-shell-linter.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -euo pipefail
+shopt -s globstar nullglob
+GLOBIGNORE="vendor*:node_modules*:.git*"
+
+declare -a excluded=(
+    'test/dockerized-integration-tests.sh'
+    'test/run-php-linter.sh'
+    'test/unittests.sh'
```

Comment:
```suggestion
```
Suggest to led by example!

Reply:
test/unittests.sh fails shellcheck, are you suggesting it be fixed with this PR?